### PR TITLE
Dist docs.update

### PR DESCRIPTION
**Pull request by flavio-fernandes**
This is a much needed update to dist-docs for easy reference to the latest man pages in ovn.
The previous update was done in for 20.03.

### CODE REVIEW

**Comment by flavio-fernandes** [comment]
@numansiddique @putnopvut pls take a peek on this when you can.

**Comment by numansiddique** [comment]
Thanks a  lot for updating this.

For a moment I thought you manually edited all the docs. :)

I agree we should automate. I was under the impression we do it already. We also have https://docs.ovn.org/en/latest/

Agree. We need to automate all of this.

This LGTM.

File: src/static/support/dist-docs/ovn-appctl.8.html [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <html><head><meta charset="UTF-8"></head><body><pre>
-ovn-appctl(8)                 Open vSwitch Manual                ovn-appctl(8)
+ovn-appctl(8)                     OVN Manual                     ovn-appctl(8)
 
 
 
+build/.PP
```
Comment (halfcrazy) [inline src/static/support/dist-docs/ovn-appctl.8.html:6]: typo?

File: src/static/support/dist-docs/ovn-appctl.8.txt [hunk 2]
```diff
@@ -1,7 +1,9 @@
-ovn-appctl(8)                 Open vSwitch Manual                ovn-appctl(8)
+ovn-appctl(8)                     OVN Manual                     ovn-appctl(8)
 
 
 
+build/.PP
```
Comment (halfcrazy) [inline src/static/support/dist-docs/ovn-appctl.8.txt:5]: ditto

**Comment by flavio-fernandes** [comment]
> Thanks a lot for updating this.
> 
> For a moment I thought you manually edited all the docs. :)
> 
> I agree we should automate. I was under the impression we do it already. We also have https://docs.ovn.org/en/latest/
> 
> Agree. We need to automate all of this.
> 
> This LGTM.

LOL. Yes, a lot of typing! ;P

So, we do have automation for the read the docs portion already (https://docs.ovn.org/en/latest/) that is hooked via
https://readthedocs.org/projects/ovn/builds/

The part that is not automated is the generated man pages. I noticed that they are referenced from the
[ovn repo itself](https://github.com/ovn-org/ovn/blob/master/Documentation/ref/index.rst), so special care must be taken not to break that.

File: src/static/support/dist-docs/ovn-appctl.8.html (hunk 1 above)
Comment (flavio-fernandes) [reply src/static/support/dist-docs/ovn-appctl.8.html:6]: hm... good catch. I think this is an issue with the dist-docs build in the core [ovn repo](https://github.com/ovn-org/ovn/blob/master/Documentation/ref/index.rst). I am particular curious about the fact that only a few of them have that.
I will have to play a little more with that in order to understand why some of the generated files
have that weird blurb on them. I wonder if @blp or @putnopvut would already know.

**Pull request merged**